### ISSUE TITLE
Rufo keeps switching formatting on same file every time it runs

### ISSUE THREAD

**Issue by ivoanjo** [ruby-formatter/rufo#86]
While looking into #83 I tried rufo on a number of repositories for commonly-used gems and came across a few issues.

This one comes from the [dry-types](https://github.com/dry-rb/dry-types) gem.

The file `to_ast_spec.rb` link <https://raw.githubusercontent.com/dry-rb/dry-types/761bc00b24dc4d4a66fd4ef5a514e72b1558d5f7/spec/dry/types/to_ast_spec.rb> keeps getting switched back and forth between two formats by rufo:

```bash
$ wget "https://raw.githubusercontent.com/dry-rb/dry-types/761bc00b24dc4d4a66fd4ef5a514e72b1558d5f7/spec/dry/types/to_ast_spec.rb"
(...)
$ rufo to_ast_spec.rb 
Format: to_ast_spec.rb
$ md5sum to_ast_spec.rb 
e50ddccef298cb1d99ea53022219dc95  to_ast_spec.rb
$ rufo to_ast_spec.rb 
Format: to_ast_spec.rb
$ md5sum to_ast_spec.rb 
525b86cbabfe431098db93cb6331f9d4  to_ast_spec.rb
$ rufo to_ast_spec.rb 
Format: to_ast_spec.rb
$ md5sum to_ast_spec.rb 
e50ddccef298cb1d99ea53022219dc95  to_ast_spec.rb
$ rufo to_ast_spec.rb 
Format: to_ast_spec.rb
$ md5sum to_ast_spec.rb 
525b86cbabfe431098db93cb6331f9d4  to_ast_spec.rb
```

**Comment by mjago** [comment]
Reduced:
```` Ruby
foo([
      [
        bar,
      ],
    ])
````

**Comment by mjago** [comment]
Fixed with #111
